File: quiz 2/quizQ12.js
// Write a program to change the capitalization of all letters in a given string.

// Possibly need to split the string into an array of letters then deal with them each individually
// Need to use toUpperCase and toLowerCase within if statements to check the current case of the letter so it can be swtiched

function switchCase(string) {
    let newString = '';
    for (var i = 0; i < string.length; i++) {
        if (65 <= string.charCodeAt(i) && string.charCodeAt(i) <= 90) {
            newString += String.fromCharCode(string.charCodeAt(i)).toLowerCase();
        } else if (97 <= string.charCodeAt(i) && string.charCodeAt(i) <= 122) {
            newString += String.fromCharCode(string.charCodeAt(i)).toUpperCase();
        } else if (32 === string.charCodeAt(i)) {
            newString += String.fromCharCode(32);
        }
    }
    return newString;
}

console.log(switchCase('I WAnt chocolate'));

File: quiz3/python/quizQ12.py
# // Write a program to change the capitalization of all letters in a given string.

# // Possibly need to split the string into an array of letters then deal with them each individually
# // Need to use toUpperCase and toLowerCase within if statements to check the current case of the letter so it can be swtiched
string = "North Carolina"

def case_switch():
    list(string)
    for x in string:
        if string.isupper():
            string.lower()
        elif string.islower():
            string.upper()
    newString = string.join("")
    print(newString)

case_switch()


# shortcut
print (string.swapcase())
